manifests/user.pp

```diff
@@ -193,6 +193,7 @@
   Boolean                              $managevim                = true,
   Enum['inclusive','minimum']          $membership               = 'minimum',
   String                               $password                 = '!!',
+  Optional[Integer[0,99999]]           $password_max_age         = undef,
```

Comment:
This should be `Optional[Accounts::User::PasswordMaxAge]    $password_max_age    = undef,`

Reply:
Good spot! I have changed this now.